spec/features/manage_subscriptions_confirmation_email_spec.rb

```diff
@@ -28,6 +28,9 @@ def and_i_enter_my_email_address
 
   def then_i_can_see_a_confirmation_email_has_been_sent_to_me
     expect(@request).to have_been_requested
-    expect(page).to have_content("We’ve sent an email to #{@email_address}")
+    expect(page).to have_content(
+      I18n.t("subscriber_authentication.request_sign_in_token.confirmation",
```

Comment:
Minor: we should use `t!` here to be sure this will fail with an exception if the translation doesn't exist.

Reply:
Done